media/css/firefox/whatsnew/whatsnew.scss

```diff
@@ -2,117 +2,115 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../../pebbles/includes/lib';
-@import '../../hubs/sections';
-@import 'header';
-@import 'send-to-device';
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
 
+@import '../../../protocol/css/includes/lib';
+@import '../../protocol/components/state-fxa';
```

Comment:
Is this import needed for this page?

Reply:
Oops, copy-pasta. There isn't any account-based conditional content on this page.